community/kernel/src/main/java/org/neo4j/kernel/impl/api/cursor/TxAbstractNodeCursor.java

```diff
@@ -135,6 +135,14 @@ public long id()
     }
 
     @Override
+    public Cursor<RelationshipItem> relationships(Direction direction, int relType)
+    {
+        return state.augmentNodeRelationshipCursor(
+                nodeIsAddedInThisTx ? Cursors.<RelationshipItem>empty() : cursor.get().relationships(direction, relType), nodeState,
+                direction, new int[]{relType} );
```

Comment:
An annoying new int[] for sure, even knowing that other creations have been avoided by avoiding varargs.